lib/install/planbuilder.go

```diff
@@ -555,19 +625,70 @@ func splitServers(servers []storage.Server, app app.Application) (masters []stor
 	return masters, nodes, nil
 }
 
-// skipDependency returns true if the dependency package specified by dep
-// should be skipped when installing the provided application
-func (b *PlanBuilder) skipDependency(dep loc.Locator) bool {
-	// rbac-app is installed separately
-	if dep.Name == constants.BootstrapConfigPackage {
-		return true
+func addResources(builder *PlanBuilder, resourceBytes []byte, runtimeResources []runtime.Object) error {
+	kubernetesResources, gravityResources, err := splitResources(resourceBytes)
+	if err != nil {
+		return trace.Wrap(err)
 	}
-	// do not install bandwagon unless the app uses it in its post-install
-	if dep.Name == defaults.BandwagonPackageName {
-		setup := b.Application.Manifest.SetupEndpoint()
-		if setup == nil || setup.ServiceName != defaults.BandwagonServiceName {
-			return true
+	if len(gravityResources) != 0 {
+		var buf bytes.Buffer
+		err = storage.Encode(gravityResources, &buf)
+		if err != nil {
+			return trace.Wrap(err)
 		}
+		builder.gravityResources = buf.Bytes()
```

Comment:
Do we not need to exclude RuntimeEnvironment resource from here as well? Won't it launch env update operation right after install otherwise?

Reply:
It won't but I will.